src/components/DocumentListItem/DocumentListItem.js

```diff
@@ -1,8 +1,7 @@
 import React, { useState } from "react";
-import "font-awesome/css/font-awesome.min.css";
 import "./DocumentListItem.css";
-import PropTypes from "prop-types";
-import { formatDate } from "./../../util/dateFormatter";
+import { formatDate } from "../../util/dates";
```

Comment:
I renamed so the file could potentially hold other date logic. And better follows convention of util files with high-level names rather than repeating the name of a single function.

Reply:
Completed